lists/global.csv

```diff
@@ -1115,5 +1115,7 @@ http://zoomshare.com/,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,
 https://www.opentech.fund/,HUMR,Human Rights Issues,2017-09-26,OONI,
 https://www.womenonwaves.org/,HUMR,Human Rights Issues,2017-10-26,OONI,
 https://www.womenonweb.org/,HUMR,Human Rights Issues,2017-10-26,OONI,
+https://www.organicbouquet.com/,CULTR,Culture,2018-02-14,OONI,Valentine's Day site (some countries ban Valentine's Day)
+https://www.hallmark.com/valentines-day/,CULTR,Culture,2018-02-14,OONI,Valentine's Day site (some countries ban Valentine's Day)
```

Comment:
Hallmark is a pretty popular retailer, yet the valentines day page is behind https, so it's also, I don't think, the most relevant thing we can add to test if content related to valentines day is blocked.